core/ibft.go

```diff
@@ -924,6 +922,8 @@ func (i *IBFT) runFin() {
 
 	// Remove stale messages
 	i.messages.PruneByHeight(i.state.getHeight())
+
+	return true
 }
 
 // moveToNewRound moves the state to the new round
```

Comment:
Change the comment not to include `move the state to the new round`

Reply:
Fixed 94978a3b28f7a8bef95d9fe77e78fea524c6c1eb